ror-tools/src/test/scala/tech/beshu/ror/tools/utils/DirectoryUtils.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.tools.utils
+
+import java.io.IOException
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.{Files, Path}
+import java.security.MessageDigest
+import scala.jdk.CollectionConverters.*
+
+object DirectoryUtils {
+
+  def clean(dir: Path): Unit = {
+    if (dir.toFile.exists()) {
+      Files.walkFileTree(dir, new java.nio.file.SimpleFileVisitor[java.nio.file.Path]() {
```

Comment:
better-files have the `delete` (with true flag as a parameter) to recursively delete folder files. Can we use it instead?